src/components/screens/monitor/blockDetails/blockDetails.css

```diff
@@ -0,0 +1,32 @@
+@import '../../../../app/variables.css';
+@import '../../../../app/mixins.css';
+
+.tableRow {
+  &:hover {
+    background-color: transparent !important;
+  }
+}
+
+.dataContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  flex-grow: 1;
+  box-sizing: border-box;
+  width: calc(100% / 5);
+
+  & > label {
+    @mixin contentNormal bold;
+  }
+
+  & > span {
```

Comment:
We agreed not to use cascading css for generic elements like `div` or `span` https://trello.com/c/J5BXlkpx

In this case, you could just move the margin to the `label` and the mixin to the `dataContainer`